cub/cub/device/device_copy.cuh

```diff
@@ -72,9 +50,9 @@ struct DeviceCopy
   //!    {
   //!      __host__ __device__ __forceinline__ auto operator()(uint32_t index)
   //!      {
-  //!        return thrust::make_constant_iterator(d_data_in[index]);
+  //!        return thrust::make_constant_iterator(d_dataccessor_in[index]);
   //!      }
-  //!      int32_t *d_data_in;
+  //!      int32_t *d_dataccessor_in;
```

Comment:
Suggestion: revert renaming the variables in the documentation for `Batched`, since they are unrelated to `mdspan` or accessors.